### DISCUSSION TITLE
video preview, default finder app, cmd-down hotkey

### DISCUSSION THREAD

**Discussion by babichfx** [mikekazakov/nimble-commander#195] [Ideas]
Hey! Overall thank you, the app is great! Using it every day!
And have some wishes, what do you think? 
- Now the video preview starting on pause so I need to use the mouse to start the video anyway. Would be great to preview with macos default preview it is more familiar on mac. 
- Would be great if the Nimble will be the default app for files operation - how it was made in "Path Finder". So 'open .' in terminal will open the Nimble not finder.
- One more wish is to setup the cmd-down for open files, but enter to rename files, because its native for macos. Now I can change it in shortcuts menu, but because of conflict the cmd-down is not working properly on files. 
 Thank you! You are doing great job!

**Comment by mikekazakov** [comment]
Hi, thanks for the feedback, my comments are below:
> Hey! Overall thank you, the app is great! Using it every day! And have some wishes, what do you think?
> 
> * Now the video preview starting on pause so I need to use the mouse to start the video anyway. Would be great to preview with macos default preview it is more familiar on Mac.

There's already an opened issue, though please look at https://github.com/mikekazakov/nimble-commander/discussions/136. If anybody has tips on how it can be implemented - please share them.

> * Would be great if the Nimble will be the default app for files operation - how it was made in "Path Finder". So 'open .' in terminal will open the Nimble not finder.

This has been already discussed, please search for the answer either here or on the old forum (https://magnumbytes.com/forum/index.php).

> * One more wish is to setup the cmd-down for open files, but enter to rename files, because its native for macos. Now I can change it in shortcuts menu, but because of conflict the cmd-down is not working properly on files.

Not sure I understand the conflict. If you have a minimal but detailed repro - go ahead and open an issue.

>   Thank you! You are doing great job!

Thanks!

**Comment by babichfx** [comment]
All the issues was resolver with time, thank you. 
For the second one - default file browser - I found the better solution than `defaults write -g NSFileViewer -string info.filesmanager.Files` : 
you need for a **duti** app, it can be installed from brew
it can change the default file manager with:
`duti -s info.filesmanager.Files public.folder all`
revert is: 
`duti -s com.apple.finder public.folder all`

works really well for me now, all the "Reveal in finder" commands, `open .` in terminal and others works good.
